client/my-sites/importer/logos/godaddy-gocentral.jsx

```diff
@@ -5,11 +5,14 @@
  */
 import React from 'react';
 
-export default function GoDaddyGoCentralLogo( { size } ) {
+export default function GoDaddyGoCentralLogo( {
+	size,
+	className = 'godaddy-gocentral social-logo importer__service-icon',
+} ) {
 	return (
 		<img
 			// eslint-disable-next-line wpcalypso/jsx-classname-namespace
```

Comment:
Do we still need the `eslint-disable-next-line` here?

Reply:
The linter is happy with it removed, so I've taken it out.